src/screens/AddSiteScreen.js

```diff
@@ -1,204 +1,339 @@
-
-                                      <Button
-                                        onPress={()=> alert('submitted')}
-                                        title="Open modal"
-                                        large
-                                        rounded={true}
-                                        buttonStyle={buttonStyle}
-                                        icon={{name: 'plus', type: 'font-awesome'}}
-                                        title={submit}
-                                      >  {submit}
-                                       </Button>
-                                    </CardSection>
-                                  </ScrollView>
-                                </Modal>
-                              </View>
-                            );
-                          }
-                        }
-
-              const styles = {
-                  descriptionStyle: {
-                      color: darkBlue,
-                      fontSize: 15
-                  },
-                  sectionStyle: {
-                      backgroundColor: grey
-                  },
-                  buttonStyle:{
-                    marginTop: 10,
-                    marginBottom: 10,
-                    backgroundColor: navyBlue
-                  },
-                  headerTitle:{
-                    flex: 1,
-                    marginTop: 20,
-                    color: navyBlue,
-                     justifyContent: 'center',
-                     alignSelf:'center',
-                  },
-                  largeTextInput: {
-                    height: 100
-                  }
-
-              }
-
-export default AddSiteScreen;
+    }
+
+    onUpdateLatitudeText = (newLatText) => {
+        this.props.updateLatitudeText({latitudeText: newLatText})
+    };
+
+    onUpdateLongitudeText = (newLongText) => {
+        this.props.updateLongitudeText({longitudeText: newLongText})
+    };
+
+    onUpdateSiteTitleText = (newTitleText) => {
+        this.props.updateSiteTitleText({siteTitleText: newTitleText})
+    };
+
+    onUpdateSiteDescriptionText = (newDescriptionText) => {
+        this.props.updateSiteDescriptionText({siteDescriptionText: newDescriptionText})
+    };
+
+    onUpdateSiteDirectionsText = (newDirectionsText) => {
+        this.props.updateSiteDirectionsText({siteDirectionsText: newDirectionsText})
+    };
+
+    onUpdateSiteNearestTownText = (newNearestTownText) => {
+        this.props.updateSiteNearestTownText({siteNearestTownText: newNearestTownText})
+    };
+
+    onUpdateAccessibilityOption = (newAccessibilityOption) => {
+        this.props.updateAccessibilityOption({accessibilityOption: newAccessibilityOption})
+    };
+
+    onUpdateFacilitiesOption = (newFacilitiesOption) => {
+        this.props.updateFacilitiesOption({facilitiesOption: newFacilitiesOption})
+    };
+
+    onClickReset = () => {
+        this.props.resetAddScreenFields();
+    };
```

Comment:
Above are all the functions that call the actions for passing along what's selected.